From d080f6dbc88efacc4017bf85853037c88cb05411 Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Sat, 22 Jun 2019 19:57:57 -0700
Subject: [PATCH 29/36] build: Use ifeq/else ifeq/else construction in board
 config

* Instead of nesting ifeq/else/ifeq/else/endif/endif construction, make
  allows the more readable ifeq/else ifeq/else/endif construction. Use it
  instead.

Change-Id: Ifd2635929799db593e04a13827f05c97e95487e4

# Conflicts:
#	config/BoardConfigQcom.mk
---
 config/BoardConfigKernel.mk |  4 +---
 config/BoardConfigQcom.mk   | 20 ++++++++++----------
 2 files changed, 11 insertions(+), 13 deletions(-)

diff --git a/config/BoardConfigKernel.mk b/config/BoardConfigKernel.mk
index 94c44ec8..1f3f8ad6 100644
--- a/config/BoardConfigKernel.mk
+++ b/config/BoardConfigKernel.mk
@@ -74,11 +74,9 @@ KERNEL_TOOLCHAIN := $(TARGET_KERNEL_TOOLCHAIN_ROOT)
 endif
 ifeq ($(KERNEL_TOOLCHAIN),)
 KERNEL_TOOLCHAIN_PATH := $(KERNEL_TOOLCHAIN_PREFIX)
-else
-ifneq ($(KERNEL_TOOLCHAIN_PREFIX),)
+else ifneq ($(KERNEL_TOOLCHAIN_PREFIX),)
 KERNEL_TOOLCHAIN_PATH := $(KERNEL_TOOLCHAIN)/$(KERNEL_TOOLCHAIN_PREFIX)
 endif
-endif
 
 ifneq ($(USE_CCACHE),)
     # Detect if the system already has ccache installed to use instead of the prebuilt
diff --git a/config/BoardConfigQcom.mk b/config/BoardConfigQcom.mk
index 70cdf679..25392c95 100644
--- a/config/BoardConfigQcom.mk
+++ b/config/BoardConfigQcom.mk
@@ -61,24 +61,19 @@ MASTER_SIDE_CP_TARGET_LIST := msm8996 msm8998 sdm660 sdm845 sm6150 sm8150
 ifeq ($(call is-board-platform-in-list, $(B_FAMILY)),true)
     MSM_VIDC_TARGET_LIST := $(B_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8974
-else
-ifeq ($(call is-board-platform-in-list, $(B64_FAMILY)),true)
+else ifeq ($(call is-board-platform-in-list, $(B64_FAMILY)),true)
     MSM_VIDC_TARGET_LIST := $(B64_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8994
-else
-ifeq ($(call is-board-platform-in-list, $(BR_FAMILY)),true)
+else ifeq ($(call is-board-platform-in-list, $(BR_FAMILY)),true)
     MSM_VIDC_TARGET_LIST := $(BR_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8916
-else
-ifeq ($(call is-board-platform-in-list, $(UM_3_18_FAMILY)),true)
+else ifeq ($(call is-board-platform-in-list, $(UM_3_18_FAMILY)),true)
     MSM_VIDC_TARGET_LIST := $(UM_3_18_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8996
-else
-ifeq ($(call is-board-platform-in-list, $(UM_4_4_FAMILY)),true)
+else ifeq ($(call is-board-platform-in-list, $(UM_4_4_FAMILY)),true)
     MSM_VIDC_TARGET_LIST := $(UM_4_4_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8998
-else
-ifeq ($(call is-board-platform-in-list, $(UM_4_9_FAMILY)),true)
+else ifeq ($(call is-board-platform-in-list, $(UM_4_9_FAMILY)),true)
     MSM_VIDC_TARGET_LIST := $(UM_4_9_FAMILY)
     QCOM_HARDWARE_VARIANT := sdm845
 else
@@ -96,6 +91,11 @@ endif
 endif
 endif
 
+PRODUCT_SOONG_NAMESPACES += \
+    hardware/qcom/audio-caf/$(QCOM_HARDWARE_VARIANT) \
+    hardware/qcom/display-caf/$(QCOM_HARDWARE_VARIANT) \
+    hardware/qcom/media-caf/$(QCOM_HARDWARE_VARIANT)
+
 # QCOM HW crypto
 ifeq ($(TARGET_HW_DISK_ENCRYPTION),true)
     TARGET_CRYPTFS_HW_PATH ?= vendor/qcom/opensource/cryptfs_hw
-- 
2.17.1

